Bottom: 54c2913e917e4c069845e7cff6a4f6b4036dd567
Top:    9d2dfce0840ad54e812f72135522ebf9cecfa52b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 16:08:09 -0500

Refresh of asset-host

---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg b/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
index 8dc2a29b..ee683c3d 100644
--- a/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
+++ b/docker/buildbot-worker-ptxdist/buildbot/.pydistutils.cfg
@@ -1,2 +1,2 @@
 [easy_install]
-index-url=http://172.16.103.72/repository/PyPi_Proxy/simple
+index-url=http://nexus.novatech-llc.com/repository/pypi/simple
